### PR TITLE
feat: success consultation page

### PR DESCRIPTION
**Pull request by maulanayuseph**
**Task**
[[Form Pemohon] Hal 5 Revamp: Ending Form](https://app.clickup.com/t/2nxaaz7)

**Changes**
- Create component success consultation
- Handle `button` submit data

**Preview**
screencapture-localhost-8083-consultation-request-2022-03-08-12_44_35

**Evidence**
title: feat: success consultation page
project: Pikobar Website
participants: @maulanayuseph @Ibwedagama @bangunbagustapa @yoslie @doohanas

### CODE REVIEW

**Review by bangunbagustapa — APPROVED** [review]

File: components/_pages/index/ConsultationVitamin/SuccessPage.vue [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    />
+  </div>
+</template>
+
+<script>
+import { mapState } from 'vuex'
+import { faInfoCircle } from '@fortawesome/free-solid-svg-icons'
+import { consultationLink } from '../IsolasiMandiri/backlinks'
+import BaseAlert from '@/components/Base/Alert'
+import Progress from '~/components/_pages/index/ConsultationVitamin/ProgressHeader.vue'
```
Comment (Ibwedagama) [inline components/_pages/index/ConsultationVitamin/SuccessPage.vue:67]: hello @maulanayuseph , I think we should stick with one way to import, either using the relative path or using an alias, wdyt? 🤔

**Review by Ibwedagama — APPROVED** [review]

**Review by yoslie — APPROVED** [review]

File: components/_pages/index/ConsultationVitamin/SuccessPage.vue (hunk 1 above)
Comment (maulanayuseph) [reply components/_pages/index/ConsultationVitamin/SuccessPage.vue:67]: Hi @Ibwedagama you mean like this
```
import { consultationLink } from '~/components/_pages/index/ConsultationVitamin/IsolasiMandiri/backlinks'
import BaseAlert from '~/components/Base/Alert'
```

Comment (Ibwedagama) [reply components/_pages/index/ConsultationVitamin/SuccessPage.vue:67]: yea I think it's better to choose one for consistency, either use `~`, `@` or relative path

Comment (maulanayuseph) [reply components/_pages/index/ConsultationVitamin/SuccessPage.vue]: Thanks fixed on this [commit](https://github.com/jabardigitalservice/pikobar-jabarprov-go-id/pull/479/commits/670503d080e36a14711a38a130e7655a3fb08532)

**Review by doohanas — APPROVED** [review]

**Review by naufalihsank — APPROVED** [review]

**Pull request merged**